src/cpp/communication/Track.cpp

```diff
@@ -150,15 +150,15 @@ void Track::transmit_()
     {
         // Get data received
         std::unique_ptr<DataReceived> data;
-        ReturnCode take_result = reader_->take(data);
+        ReturnCode ret = reader_->take(data);
```

Comment:
Why changing this variable name? I would have left it as `take_result` or `take_return`

Reply:
Because it is being reused below for write.